Extensions/old_stats.js

```diff
@@ -1,108 +1,49 @@
 //* TITLE Old Stats **//
-//* VERSION 0.2.2 **//
+//* VERSION 0.3.0 **//
 //* DESCRIPTION  **//
 //* DEVELOPER STUDIOXENIX **//
 //* FRAME false **//
 //* BETA false **//
 
 XKit.extensions.old_stats = new Object({
 
-	running: false,
+    running: false,
+
+    preferences: {
+        "sep-0": {
+            text: "Old Stats is now Old Sidebar",
+            type: "separator"
+        },
+    },
+
+    run: function() {
+        this.running = true;
+				if (!XKit.installed.check("old_sidebar")) {
+	        XKit.install("old_sidebar", function(mdata) {
```

Comment:
the extension isn't called old_sidebar, it's called "estufars_sidebar_fix" so this doesn't work at all.